### ISSUE TITLE
Needs Win/Super key modifier

### ISSUE THREAD

**Issue by rbreaves** [gileli121/WindowTop#100]
Love the program, just started using it and that shrink option is something I've wanted but didn't know was even possible. At any rate I use my Kinto app with this https://github.com/rbreaves/kinto and it appears to get confused a little bit. Detects the actual physical key instead of the mapped ones, but not a big deal - until I saw you only have Ctrl and Alt as mappable options. My Win key behaves as my Alt key in practically all cases and I would need Win/Super to be an option if I were to use my virtual Alt key.

As it stands now though when I run Kinto this app conflicts with my Ctrl based modifiers so simple things like like Ctrl-A to select all activates default actions in this app instead because it sees Alt-A.

You don't need to make it play nice with Kinto - it will do that already if it supports Win/Super.. however confusing it may be to some users lol.

**Comment by gileli121** [comment]
@rbreaves 

> until I saw you only have Ctrl and Alt as mappable options

This is not true.
You can use Shift key also
![image](https://user-images.githubusercontent.com/17680514/120024690-3bbd7a00-bff8-11eb-88a3-0b64fd496c46.png)

**Comment by rbreaves** [comment]
Shift is not the same type of modifier key as Ctrl, Alt, and Win. I was obviously referring to system level modifiers that are similar to function of each other as far as being on the bottom row and modifying system level behavior. I do not care about Shift, Caplock, scroll lock, etc.

And yes I would expect that you had support for Shift as it is commonly used for additional combo layers involving - system level modifier keys. Again I would love to use this application but only supporting 2 of the system level shortcut keys is an issue for me and potentially other users that make heavy use of all 3 modifier keys (aka professional mac users and developers).

**Comment by gileli121** [comment]
@rbreaves 
The last system-level modifier is the Win key (The key that opens the start menu)?
All you want is that it will support also the Win key right?

**Comment by rbreaves** [comment]
@gileli121 that is correct, I know it’s typically used for the start menu but I go through efforts to disable that start menu activation & other win key combos & I replace them w/ my own.

why I’d find it useful, so I can leverage that key still w/ Window top. I align many hotkeys w/ macs as much as I can.

**Comment by gileli121** [comment]
OK, I will implement it in v5.6 or v5.7

**Comment by koppor** [comment]
My use case is to have <kbd>Ctrl</kbd> as second modifier key.

![grafik](https://user-images.githubusercontent.com/1366654/148408030-4edc9ada-4bdd-4932-8629-fd3ca04cc425.png)

I tried to set <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>A</kbd> as hotkey for "Set Click-Through mode for the currently active window", but the box did not accept <kbd>Ctrl</kbd>.

I am running Version v5.7.7.
